### PR TITLE
HTML/CSS Basic Web App Initialized with linters

### PR DESCRIPTION
**Pull request by Markotweyo**
-Added HTML file
-Added CSS file
-Added yml file for html/css linter in github workflow
-Added .gitignore file for node_modules folder and /test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+
+> Hello World Microverse Project.
+
+
+## Built With
+
+- HTML/CSS
+- linters
+
+
```
Comment (danmainah) [inline README.md:12]: - [ ] Kindly add the getting started section, you can use [this template](https://github.com/microverseinc/readme-template) for guide you in implementing it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Markotweyo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

# Required Changes ♻️

- [ ] Kindy make sure your pull request title is descriptive, currently `Linters` used is not descriptive.
- [ ] Kindly make sure your pull request summary is described, you could indicate in point form what you did in the pull request like setup linters, add html file etc.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+# Project Name
+
+> Hello World Microverse Project.
+
```
Comment (Markotweyo) [inline README.md:5]: Thank you for the review. I have fixed the project description.

File: README.md (hunk 1 above)
Comment (Markotweyo) [inline README.md:13]: I have fixed the README section according to the template

File: index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Hello Microverse</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles.css">
+</head>
```
Comment (ishimwezachee) [inline index.html:11]: - [ OPTIONAL]  Your path to your styling file has a small issue, which is breaking styling to appear on your web page. You can use something like `  <link rel="stylesheet" href="styles.css">` or add the dot before your slash `./`

**Review by ishimwezachee — APPROVED** [review]
Hi  @Markotweyo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

- I am sorry for these were not mentioned by the previous reviewer, but they are very important for you to fulfill the requirements of your project. 

- [ OPTIONAL] connect your CSS and HTML correctly 

- [ OPTIONAL] your branch name is not good, I recommend to have descriptive names like :   (e.g. 'created_sidebar' or 'added_comments_section').

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**